quickref/qsharp-quick-reference.tex

```diff
@@ -139,14 +139,24 @@
   \end{keysref}
 
   \begin{keysref}{Derived Types}
-    Arrays                  & \texttt{\emph{elementType}[]} \\
+    Arrays                  & \texttt{\emph{elementType}[]} \newline
+    							\texttt{(elementType,elementType)[][] } \\
```

Comment:
I did not know that all spaces show the same way in all editors!

Reply:
They do (well, some editors highlight them but at least they always occupy a single space). Tabs show differently - GitHub shows them as taking 8 spaces, and other editors typically use 2 or 4 spaces, so it can be less noticeable in local editor but very noticeable in a pull request on GitHub.